src/lib/createables/misc.ts

```diff
@@ -0,0 +1,98 @@
+import itemID from '../util/itemID';
+import { Bank } from '../types';
+
+interface Createable {
+	name: string;
+	outputItems: Bank;
+	inputItems: Bank;
+	smithingLevel?: number;
+	addOutputToCollectionLog?: boolean;
```

Comment:
i think it would be best if you reverted your refactors to creatables, and just add these items like normal for now, even if the file is very big, i think it should be refactored ina  different PR, nothing unrelated to prayer, unless its needed, should be in it.